### PR TITLE
New SiriusImageView and SiriusCameraView widget

### PR DESCRIPTION
**Pull request by anacso17**
This PR adds new version of SiriusImageView and adds a first simple version of SiriusCameraView widget. 
I don't know if these names are appropriate, so suggestions are welcome.

There is also a commit that adapts the Transport Line Control HLAs to use SiriusCameraView widget.

### CODE REVIEW

**Comment by anacso17** [comment]
Thank you @fernandohds564! I made some changes following your suggestions!

**Comment by anacso17** [comment]
@fernandohds564 I made some changes in the widgets appearance and adapted my windows. 
Please take a look, if you have time :)

File: pyqt-apps/siriushla/widgets/scrn_view.py [hunk 1]
```diff
@@ -0,0 +1,828 @@
...
+                self.image_view.image_maxheight)
+        self.PyDMSpinbox_ROIHeight.send_value()
+        self.PyDMSpinbox_ROIWidth.value_changed(
+                self.image_view.image_maxwidth)
+        self.PyDMSpinbox_ROIWidth.send_value()
+        self.PyDMSpinbox_ROIOffsetX.value_changed(0)
+        self.PyDMSpinbox_ROIOffsetX.send_value()
+        self.PyDMSpinbox_ROIOffsetY.value_changed(0)
+        self.PyDMSpinbox_ROIOffsetY.send_value()
+        time.sleep(0.3)
```
Comment (fernandohds564) [inline pyqt-apps/siriushla/widgets/scrn_view.py]: Hi @anacso17!
I think this waiting timing may not be enough. If we operate the camera in the external trigger mode, images will only be available at 2Hz.

File: pyqt-apps/siriushla/widgets/scrn_view.py [hunk 2]
```diff
@@ -0,0 +1,828 @@
...
+        self.image_view.saveCalibrationGrid()
+        time.sleep(0.3)
```
Comment (fernandohds564) [inline pyqt-apps/siriushla/widgets/scrn_view.py]: Do you think it is needed to wait here? I think we could drop this...

File: pyqt-apps/siriushla/widgets/scrn_view.py [hunk 3]
```diff
@@ -0,0 +1,828 @@
...
+        self.PyDMSpinbox_ROIHeight.value_changed(int(roi_h))
+        self.PyDMSpinbox_ROIHeight.send_value()
+        self.PyDMSpinbox_ROIWidth.value_changed(int(roi_w))
+        self.PyDMSpinbox_ROIWidth.send_value()
+        self.PyDMSpinbox_ROIOffsetX.value_changed(int(roi_offsetx))
+        self.PyDMSpinbox_ROIOffsetX.send_value()
+        self.PyDMSpinbox_ROIOffsetY.value_changed(int(roi_offsety))
+        self.PyDMSpinbox_ROIOffsetY.send_value()
+        time.sleep(0.2)
```
Comment (fernandohds564) [inline pyqt-apps/siriushla/widgets/scrn_view.py]: Maybe `0.2` is too much. I think 0.05 is more than enough. What do you think?

File: pyqt-apps/siriushla/widgets/scrn_view.py (hunk 2 above)
Comment (anacso17) [reply pyqt-apps/siriushla/widgets/scrn_view.py]: I agree, I will drop this!

File: pyqt-apps/siriushla/widgets/scrn_view.py (hunk 3 above)
Comment (anacso17) [reply pyqt-apps/siriushla/widgets/scrn_view.py]: I tried to keep this interval small but I had seg fault even with 0.05. 
I will reduce this to 0.1!

File: pyqt-apps/siriushla/widgets/scrn_view.py (hunk 1 above)
Comment (anacso17) [reply pyqt-apps/siriushla/widgets/scrn_view.py]: As you suggested, I implemented a "wait for max size image" in _SiriusImageView.saveCalibrationGrid, where I raise a warning dialog if could not save the grid.
I think this is a good strategy. Thank you!

**Comment by anacso17** [comment]
@fernandohds564 yes! There will be PVs to control acquisition mode (automatic or triggered) and acquisition time (when automatic). I am  including a detail control window for the camera device for these and other PVs!

**Pull request merged**